ansible/cluster-setup/README.md

```diff
@@ -32,67 +30,98 @@ A guide to installing `Assemble with Ansible`
-1. In order to create a webhook you'll need to Create a new GitHub Application.  You can do so by running the following command:
+## Configuration
+
+For ease of setup, set the `OPENSHIFT_CLUSTER_INFO` variable for use later.
+
+``` sh
+export OPENSHIFT_CLUSTER_INFO=$(echo "$(oc cluster-info | grep -Eo '.cluster(.*?).com')")
+```
 
-### Linux
+If you are using Linux environment, set the alias for the following commands to work:
 
-```bash
-xdg-open "https://github.com/organizations/$GITHUB_ORGANIZATION/settings/apps/new?url=https://janus-idp.io/blog&webhook_active=false&public=false&administration=write&checks=write&actions=write&contents=write&statuses=write&vulnerability_alerts=write&dependabot_secrets=write&deployments=write&discussions=write&environments=write&issues=write&packages=write&pages=write&pull_requests=write&repository_hooks=write&repository_projects=write&secret_scanning_alerts=write&secrets=write&security_events=write&workflows=write&webhooks=write"
+``` sh
+alias open="xdg-open"
+```
+### Set Personal Access Token
+Generate a [Personal Access Token](https://github.com/settings/tokens) for GitHub and set the `GITHUB_TOKEN` environment variable. See the official [Backstage Documentation](https://backstage.io/docs/getting-started/configuration#setting-up-a-github-integration) for information on how to create one.
+``` sh
+export GITHUB_TOKEN=
 ```
 
-### macOS
+### Create GitHub Organization
+
+Create a new [Github Organization](https://github.com/account/organizations/new?plan=free). This organization will be where the code repositories for the Components created in Backstage will go.
+
+The `GITHUB_ORGANIZATION` environment variable will be set to the name of the Organization. You may also use any organization you are a member of, as long as you have the ability to create new repositories within it.
+
+``` sh
+export GITHUB_ORGANIZATION=
+```
+
+### Set Up GitHub Application
+
+Create a new GitHub Application to enable the creation of Git WebHooks by the demo.  The required field will be populated, and correct permissions set.
 
-```bash
-open "https://github.com/organizations/$GITHUB_ORGANIZATION/settings/apps/new?url=https://janus-idp.io/blog&webhook_active=false&public=false&administration=write&checks=write&actions=write&contents=write&statuses=write&vulnerability_alerts=write&dependabot_secrets=write&deployments=write&discussions=write&environments=write&issues=write&packages=write&pages=write&pull_requests=write&repository_hooks=write&repository_projects=write&secret_scanning_alerts=write&secrets=write&security_events=write&workflows=write&webhooks=write"
+``` sh
+open "https://github.com/organizations/$GITHUB_ORGANIZATION/settings/apps/new?name=$GITHUB_ORGANIZATION-tekton-webook&url=https://janus-idp.io/blog&webhook_active=false&public=false&administratio
```

Comment:
This command has some line breaks in it so it is not copying correctly

Reply:
Good catch, just pushed a fix to that